docs/adv/configure-alexa-service.rst

```diff
@@ -0,0 +1,18 @@
+Configuring the Alexa Service
+=============================
+
+There are a few steps to enable the Alexa service on a custom NetsBlox deployment. They are outlined here:
+
+1. Register "Amazon Alexa" as an OAuth client using the NetsBlox CLI
+2. Generate LWA Keys (instructions `here <https://www.npmjs.com/package/ask-smapi-sdk>`_)
```

Comment:
The link here would probably want two underscores like \`here <...>\`__. If there's only one, it'll make a symbol table mapping link text to target, so if we ever add another link (on this page) with text `here` it'll give an ambiguous target error/warning.